sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryExecutionErrors.scala

```diff
@@ -1907,8 +1906,7 @@ object QueryExecutionErrors extends QueryErrorsBase {
   def aesModeUnsupportedError(mode: String, padding: String): RuntimeException = {
     new SparkRuntimeException(
       errorClass = "UNSUPPORTED_FEATURE",
-      messageParameters = Array(
-        s"AES-$mode with the padding $padding by the $aesFuncName function."))
+      messageParameters = Array("AES_MODE", s"$mode", s"$padding", s"$aesFuncName"))
```

Comment:
```suggestion
      messageParameters = Array("AES_MODE", mode, padding, aesFuncName))
```

Reply:
BTW, shall we quote the function name?

Reply:
They have been already quoted, see the definition of `aesFuncName `:
```scala
  private val aesFuncName = toSQLId("aes_encrypt") + "/" + toSQLId("aes_decrypt")
````